### PR TITLE
Moving condition service over to typescript class

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/services/conditionsService.ts [hunk 1]
```diff
@@ -5,8 +5,14 @@ import * as builder from './licenceWithConditionsBuilder'
 import { getAdditionalConditionsConfig, standardConditions, CURRENT_CONDITION_VERSION } from './config/conditionsConfig'
 import { AdditionalConditions, Licence } from '../data/licenceTypes'
 
-export default function createConditionsService() {
-  function getFullTextForApprovedConditions(licence: Licence) {
+export default class ConditionsService {
```
Comment (andrewrlee) [inline server/services/conditionsService.ts:8]: moving to class so get better type info than a factory method returning a dynamic object

File: test/routes/contact.test.ts [hunk 2]
```diff
@@ -12,13 +12,13 @@ jest.mock('../../server/services/userAdminService')
...
 describe('/contact', () => {
-  let userAdminService: UserAdminService
-  let roService: RoService
+  let userAdminService: jest.Mocked<UserAdminService>
+  let roService: jest.Mocked<RoService>
   let signInService
 
   beforeEach(() => {
-    roService = new RoService(undefined, undefined)
-    ;(jest as any).mocked(roService).findResponsibleOfficer.mockResolvedValue({
```
Comment (andrewrlee) [inline test/routes/contact.test.ts:21]: don't need to cast everywhere.

File: test/routes/vary.test.ts [hunk 3]
```diff
@@ -1,4 +1,3 @@
-import { mocked } from 'ts-jest/utils'
```
Comment (andrewrlee) [inline test/routes/vary.test.ts:1]: `ts-jest/utils` now spews out a load of warnings saying that its deprecated and to move over to a new version of jest. 

This just isn't needed though and it's generally cleaner to not used `mocked` at all. 

The rest of the changes in this PR are to remove usage of `mocked`

**Review by connormaglynn — APPROVED** [review]
lgtm 👍

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**